### ISSUE TITLE
start with openscm-units or scmdata.units?

### ISSUE THREAD

**Issue by MichaelTiemannOSC** [openscm/openscm-units#31]
I'm working on integrating pint into project related to climate change (via corporate disclosures).  My path was:

1. https://github.com/IAMconsortium/units/issues/9https://github.com/IAMconsortium/units/issues/9
2. https://github.com/IAMconsortium/units/files/4315212/illustrate-units.txt
3. https://github.com/openscm/scmdata
4. https://github.com/openscm/openscm-units

From what I can tell, the units handling in scmdata is really specialized to the database case, and I should be looking at openscm-units for my use case.  Can you tell me yes or no and send me on my way?  Who knows, I may be able to contribute in the future (at least to the docs so that others don't suffer the confusion that I did).

**Comment by mikapfl** [comment]
Hi Michael,

Nice to hear that you are also considering pint and maybe openscm-units for unit handling!

I was in a similar position like you are in last year when looking at how to integrate unit handling in [primap2](https://primap2.readthedocs.io/). We decided for our use case to use `openscm-units` because that gave us more flexibility  in our other choices. Mostly because `openscm-units` is very focused on solving the unit handling problem: it doesn't force you to use a specific other data handling library (besides pint). For us, this meant we could use other criteria to decide between using `pandas` and `xarray`, for example.

For us, using `openscm-units` has worked very well the past year, and we also contributed some fixes and additional functionality to `openscm-units`. I can certainly recommend using it for integrating climate-related unit handling into a python project!

Cheers,

Mika Pflüger
